diff -Nru net-tools-1.60.old/config.h net-tools-1.60.new/config.h
--- net-tools-1.60.old/config.h	2012-04-13 15:34:39.000000000 +0200
+++ net-tools-1.60.new/config.h	2015-04-10 11:57:24.820859358 +0200
@@ -49,7 +49,7 @@
 #define HAVE_HWSLIP 1
 #define HAVE_HWPPP 1
 #define HAVE_HWTUNNEL 1
-#define HAVE_HWSTRIP 1
+#define HAVE_HWSTRIP 0
 #define HAVE_HWTR 1
 #define HAVE_HWAX25 1
 #define HAVE_HWROSE 0
diff -Nru net-tools-1.60.old/config.in net-tools-1.60.new/config.in
--- net-tools-1.60.old/config.in	2000-05-21 16:32:12.000000000 +0200
+++ net-tools-1.60.new/config.in	2015-04-10 11:55:09.412341218 +0200
@@ -68,7 +68,7 @@
 bool 'SLIP (serial line) support' HAVE_HWSLIP y
 bool 'PPP (serial line) support' HAVE_HWPPP y
 bool 'IPIP Tunnel support' HAVE_HWTUNNEL y
-bool 'STRIP (Metricom radio) support' HAVE_HWSTRIP y
+bool 'STRIP (Metricom radio) support' HAVE_HWSTRIP n
 bool 'Token ring (generic) support' HAVE_HWTR y
 bool 'AX25 (packet radio) support' HAVE_HWAX25 y
 bool 'Rose (packet radio) support' HAVE_HWROSE n
diff -Nru net-tools-1.60.old/config.make net-tools-1.60.new/config.make
--- net-tools-1.60.old/config.make	2012-04-13 15:34:39.000000000 +0200
+++ net-tools-1.60.new/config.make	2015-04-10 11:58:30.524993249 +0200
@@ -16,7 +16,7 @@
 HAVE_HWSLIP=1
 HAVE_HWPPP=1
 HAVE_HWTUNNEL=1
-HAVE_HWSTRIP=1
+HAVE_HWSTRIP=0
 HAVE_HWTR=1
 HAVE_HWAX25=1
 # HAVE_HWROSE=0
