--- a/hostname.c	2001-04-08 19:04:23.000000000 +0200
+++ b/hostname.c	2014-09-18 11:01:24.312916709 +0200
@@ -30,6 +30,7 @@
  *              Foundation;  either  version 2 of the License, or  (at
  *              your option) any later version.
  */
+#include <stdlib.h>
 #include <stdio.h>
 #include <unistd.h>
 #include <getopt.h>
@@ -98,6 +99,7 @@
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
 	default:
+	    break;
 	}
 	exit(1);
     };
@@ -117,6 +119,7 @@
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
 	default:
+	    break;
 	}
 	exit(1);
     };
@@ -174,6 +177,7 @@
 	printf("%s\n", hp->h_name);
 	break;
     default:
+	break;
     }
 }
 
